hawkular-agent-core/src/main/java/org/hawkular/agent/monitor/service/AgentCoreEngine.java

```diff
@@ -573,7 +573,6 @@ private void startStorageAdapter() throws Exception {
         this.storageAdapter.initialize(
                 feedId,
                 configuration.getStorageAdapter(),
-                configuration.getGlobalConfiguration().getAutoDiscoveryScanPeriodSeconds(),
```

Comment:
why are you getting rid of this? The agent needs to scan for new discovery periodically throughout its lifetime. We can't have it just do discovery at startup and never do it again. It has to be done every X seconds so we can detect resources coming up and going away (and tell inventory when that happens).

Reply:
Iirc I added this to anticipate the metrics TTL, but there's no reason to keep it passed to AsyncInventoryStorage constructor. So I just removed it from there and mechanically removed in call hierarchy, but it's still there for the discovery job.

Reply:
oh, i see. I isn't removed, just not passed to this object. I get it now.